### PR TITLE
Fixed issue where the estimator was printing beyond the dataset size …

### PR DESCRIPTION
**Pull request by piyushghai**
…for the last batch

## Description ##
As title.

@roywei @abhinavs95 @karan6181

### CODE REVIEW

**Comment by piyushghai** [comment]
@mxnet-label-bot Add [Gluon]

File: python/mxnet/gluon/estimator/estimator.py [hunk 1]
```diff
@@ -242,7 +242,9 @@ def fit(self, train_data,
                     self.train_stats['batch_' + loss_metric.name] = loss_metric.get()[1]
 
                 try:
-                    self.train_stats['step'] = "{}/{}".format(batch_size * (i + 1), len(train_data._dataset))
+                    completed_samples = len(train_data._dataset) if i == len(train_data._dataset) - 1 \
```
Comment (roywei) [inline python/mxnet/gluon/estimator/estimator.py]: a comment for readability?

Comment (piyushghai) [reply python/mxnet/gluon/estimator/estimator.py]: Done. 568d5dfaa260a8e3c07bc36c53ae334fc7c64743

**Comment by karan6181** [comment]
Thank you for your contribution. LGTM

**Comment by piyushghai** [comment]
@mxnet-label-bot Update [Gluon, pr-awaiting-merge]

**Comment by wkcn** [comment]
Merged. Thanks for your contribution!

**Comment by wkcn** [comment]
Is it merged into the branch fit-api?

**Comment by szha** [comment]
@wkcn yes, that feature is being developed in the feature branch.

**Pull request merged**